build.cake

```diff
@@ -43,42 +44,48 @@ var informationalVersion = gitVersion.InformationalVersion;
 var buildVersion = gitVersion.FullBuildMetaData;
 var nugetversion = Argument<string>("packageVersion", gitVersion.NuGetVersion);
 
-var ANDROID_HOME = EnvironmentVariable ("ANDROID_HOME") ?? 
```

Comment:
@rmarinho all these cake changes just make this cake script consistent with master